### ISSUE TITLE
[iOS][Accessibility Issue] Adaptive card button controls are not adopting larger text setting

### ISSUE THREAD

**Issue by samyuktha24** [microsoft/AdaptiveCards#7814]
### Target Platforms

iOS

### SDK Version

2.8.5

### Application Name

Microsoft Teams

### Problem Description

On Microsoft Teams iOS platform, the adaptive card button controls are not adopting to larger text setting

- Pre-Requisites:

1. login to Teams with valid credentials
2. Font size set up to max. (Settings->Display & Text Size-> Select Larger text button as on)

- Repro Steps:

1. Open Teams application
2. Navigate over the 'Teams' and activate it.
3. Navigate over Teams and Channel and open any channel.
4. Navigate on to adaptive card.
5. Observe that adaptive card button controls are not adopting larger text setting.
 
Actual Results:
Adaptive card button controls are not adopting larger text setting.

Expected Results:
Adaptive card button controls should be adopting a larger text setting. Text size should be increased according to resize settings.

User experience:
Low vision application users who are dependent on large text sizes to read the content may get affected if the content is not visible properly in large text resize.

### Screenshots

![image](https://user-images.githubusercontent.com/9786445/188792038-70640724-3989-4808-bb6b-82ff34f85693.png)

### Card JSON

```JSON
{
        "type": "AdaptiveCard",
        "body": [
            {
                "color": null,
                "horizontalAlignment": null,
                "isSubtle": false,
                "maxLines": 0,
                "size": "medium",
                "text": "Happy Testing",
                "weight": "bolder",
                "wrap": false,
                "separator": false,
                "type": "TextBlock"
            },
            {
                "color": null,
                "horizontalAlignment": null,
                "isSubtle": false,
                "maxLines": 0,
                "size": "medium",
                "text": "Refreshed Card",
                "weight": "bolder",
                "wrap": false,
                "separator": false,
                "type": "TextBlock"
            },
            {
                "actions": [
                    {
                        "verb": "initiateSSO",
                        "title": "SSO Button",
                        "type": "Action.Execute"
                    },
                    {
                        "verb": "initiateOAuth",
                        "title": "OAuth Button",
                        "type": "Action.Execute"
                    },
                    {
                        "verb": "loopSSO",
                        "title": "Loop SSO",
                        "type": "Action.Execute"
                    },
                    {
                        "verb": "loopOAuth",
                        "title": "Loop OAuth",
                        "type": "Action.Execute"
                    },
                    {
                        "verb": "latency",
                        "title": "Latency",
                        "type": "Action.Execute"
                    },
                    {
                        "verb": "TestPreConditionFailed",
                        "title": "Test PreCondition Failed SSO",
                        "type": "Action.Execute"
                    },
                    {
                        "verb": "TestPreConditionFailedWithoutSignIn",
                        "title": "Test PreCondition Failed SSO without SignIn",
                        "type": "Action.Execute"
                    },
                    {
                        "title": "Action Submit Button",
                        "type": "Action.Submit"
                    }
                ],
                "separator": false,
                "type": "ActionSet"
            },
            {
                "color": null,
                "horizontalAlignment": null,
                "isSubtle": false,
                "maxLines": 0,
                "size": "medium",
                "text": "SSO success",
                "weight": "bolder",
                "wrap": false,
                "separator": false,
                "type": "TextBlock"
            }
        ],
        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
        "version": "1.2",
        "refresh": {
            "action": {
                "verb": "basicRefresh",
                "title": "RefreshTitle",
                "type": "Action.Execute"
            }
        }
    }
```

**Comment by paulcam206** [comment]
We fixed this on our side (tracked in bug #5283), but it looks like Teams needs to update their custom button rendering. 

@jwoo-msft - they might benefit from your insights here :)

**Comment by Wajeed-msft** [comment]
This issue has been copied over to Microsoft Teams repo here: https://github.com/MicrosoftDocs/msteams-docs/issues/6933
